lib/scripts/collections_tags_and_mesh.rb

```diff
@@ -0,0 +1,38 @@
+require 'csv'
+
+class CollectionsTagsAndMeshList
+  attr_accessor :csv_file
+
+  def initialize
+    @csv_file = File.new("#{Rails.root}/lib/scripts/results/collections_tags_and_mesh_list.csv", "w+")
+  end
+
+  def populate_collections_tags_and_mesh_info
+    CSV.open(self.csv_file.path, "w") do |csv|
+      # add headers
+      csv << ['Title', 'Link', 'Tags', 'MeSH']
+
+      select_all_collections.each do |collection|
+        # remove last "_" part of string
+        collection_data = collection.each.map {|k, v| [k.gsub(/_[a-z]+\z/, ''), v] }.to_h
+
+        csv << [clean_up_data(collection_data, "title"),
+                "https://digitalhub.northwestern.edu/collections/#{collection_data['id']}",
```

Comment:
It might make sense to put this into a variable naming the concept e.g `digital_hub_url = "https://digitalhub.northwestern.edu/collections/#{collection_data['id']}"` and then `csv << [..., digital_hub_url, ...]` . Not a big deal either way.

Reply:
Sure, I can make this change.